api/tools/src/main/java/org/pmiops/workbench/tools/DataDictionary.kt

```diff
@@ -0,0 +1,3 @@
+package org.pmiops.workbench.tools
+
+internal class DataDictionary(val meta_data: Array<Metadata>, val transformations: Array<Transformations>)
```

Comment:
I dunno whether we have strong style opinions on stuff like this but I would much rather see each arg on its own line, it's easier for me to parse that way

Reply:
nit: also, metadata should be all one word. Is the snake case because that is what Kotlin prefers for variables?

Reply:
Do you know if the spotless linter works on kotlin files? If so, this is the line format is probably enforced by spotless. If not, we should probably find and decide on a kotlin linter.

Reply:
it does! I added one. thanks for the suggestion